frontend/src/Components/NewRunConfig/NewRunState.tsx

```diff
@@ -0,0 +1,32 @@
+import { showToast } from "../Toaster";
+import AdvancedSettingsMenu from "./AdvancedSettings/AdvancedSettings";
+import NewRun from "./Settings/Settings";
+import InitConfigState from "./SimConfig";
+
+function request(configJSON: string){
+    const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: configJSON
+    };
+    var host = window.location.protocol + "//" + window.location.host;
+    fetch(`${host}/simulate`, requestOptions)
```

Comment:
Can add a .catch and if backend responds with an error, we show it (although this can be a different PR)